src/lib/config/Intents.ts

```diff
@@ -31,12 +30,6 @@ export class Intents extends BitField<IntentsString> {
 		 * - CHANNEL_PINS_UPDATE
 		 */
 		GUILDS: 1 << 0,
-		/**
-		 * - GUILD_MEMBER_ADD
-		 * - GUILD_MEMBER_UPDATE
-		 * - GUILD_MEMBER_REMOVE
-		 */
-		GUILD_MEMBERS: 1 << 1,
```

Comment:
you dont change this file. you only have to remove the 1 line from the other file